diff -urN blam-1.6.1.orig/blam.desktop.in.in blam-1.6.1/blam.desktop.in.in
--- blam-1.6.1.orig/blam.desktop.in.in	2005-01-09 13:50:57.971683568 +0100
+++ blam-1.6.1/blam.desktop.in.in	2005-01-09 13:51:23.364823224 +0100
@@ -8,7 +8,7 @@
 StartupNotify=true
 Terminal=false
 Type=Application
-Categories=Application;Network;
+Categories=GTK;GNOME;Network;News;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=blam
 X-GNOME-Bugzilla-Component=General
