diff -ur blam-1.4.1-o/configure.in blam-1.4.1/configure.in
--- blam-1.4.1-o/configure.in	2004-09-18 11:40:06.714746000 -0600
+++ blam-1.4.1/configure.in	2004-09-18 11:46:23.980420282 -0600
@@ -65,7 +65,7 @@
 AC_SUBST(LIBBLAM_LIBS)
 
 dnl Intl
-ALL_LINGUAS="bs ca cs da de el en_CA en_GB es eu fr ga gu he hr nl no pa pt_BR pt sq sr@Latn sr sv tr uk zh_CN"
+ALL_LINGUAS="bs ca cs da de el en_CA en_GB es eu fr ga gu he hr nl nb pa pt_BR pt sq sr@Latn sr sv tr uk zh_CN"
 GETTEXT_PACKAGE=blam
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE,"$GETTEXT_PACKAGE", [Gettext package])
Only in blam-1.4.1: configure.in~
