diff -urN aa/blam-1.6.0/configure.in blam-1.6.0/configure.in
--- blam-1.6.0.orig/configure.in	2004-10-23 15:12:46.000000000 +0200
+++ blam-1.6.0/configure.in	2004-11-01 19:44:12.709044680 +0100
@@ -70,13 +70,15 @@
 
 MOZILLA=
 AC_ARG_WITH([mozilla],
-	AC_HELP_STRING([--with-mozilla@<:@=mozilla|firefox|thunderbird@:>@],
-		       [Whether to use mozilla, firefox or thunderbird gtkmozembed (default: mozilla)]),
+	AC_HELP_STRING([--with-mozilla@<:@=xulrunner|mozilla|firefox|thunderbird@:>@],
+		       [Whether to use xulrunner, mozilla, firefox or thunderbird gtkmozembed (default: xulrunner)]),
 	[MOZILLA="$withval"])
 
 dnl try to autodetect: if mozilla exists, use it, else try firefox, then thunderbird
 if test "x$MOZILLA" = "x"; then
-  if pkg-config --exists mozilla-gtkmozembed; then
+  if pkg-config --exists xulrunner-gtkmozembed; then
+    MOZILLA=xulrunner
+  elif pkg-config --exists mozilla-gtkmozembed; then
     MOZILLA=mozilla
   elif pkg-config --exists firefox-gtkmozembed; then
     MOZILLA=firefox
@@ -85,7 +87,7 @@
   else
     AC_MSG_ERROR([no mozilla installation found])
   fi
-elif test "x$with_mozilla" != "xmozilla" -a "x$with_mozilla" != "xfirefox" -a "x$with_mozilla" != "xthunderbird"; then
+elif test "x$with_mozilla" != "xxulrunner" -a "x$with_mozilla" != "xmozilla" -a "x$with_mozilla" != "xfirefox" -a "x$with_mozilla" != "xthunderbird"; then
   AC_MSG_ERROR([unknown mozilla name ($MOZILLA)])
 fi
 
@@ -94,6 +96,7 @@
 AC_MSG_RESULT([$MOZILLA])
 
 case "$MOZILLA" in
+xulrunner) min_version=1.8 flavour=toolkit ;;
 mozilla) min_version=1.7 flavour=mozilla ;;
 firefox) min_version=0.10 flavour=toolkit ;;
 thunderbird) min_version=0.8 flavour=toolkit ;;
@@ -107,7 +107,7 @@
 MOZILLA_INCLUDE_ROOT="`$PKG_CONFIG --variable=includedir $MOZILLA-gtkmozembed`"
 AC_SUBST(MOZILLA_INCLUDE_ROOT)
 
-MOZILLA_HOME="`$PKG_CONFIG --variable=libdir $MOZILLA-gtkmozembed`"
+MOZILLA_HOME="`$PKG_CONFIG --variable=libdir $MOZILLA-plugin`"
 AC_SUBST(MOZILLA_HOME)
 
 MOZILLA_PREFIX="`$PKG_CONFIG --variable=prefix $MOZILLA-gtkmozembed`"
