diff -aurN blam-1.8.2.orig/libblam/Makefile.am blam-1.8.2/libblam/Makefile.am
--- blam-1.8.2.orig/libblam/Makefile.am	2005-06-19 13:36:05.000000000 +0200
+++ blam-1.8.2/libblam/Makefile.am	2005-12-01 20:51:08.000000000 +0100
@@ -15,7 +15,7 @@
         -include $(MOZILLA_INCLUDE_ROOT)/mozilla-config.h \
 	$(WERROR)
 
-blamlibdir = $(prefix)/lib/blam
+blamlibdir = $(libdir)
 blamlib_LTLIBRARIES = libblam.la
 
 libblam_la_SOURCES =				\
