diff -ur blam-1.2.4-orig/configure.in blam-1.2.4/configure.in
--- blam-1.2.4-orig/configure.in	2004-08-19 16:06:21.000000000 -0600
+++ blam-1.2.4/configure.in	2004-08-20 22:59:29.561774702 -0600
@@ -66,7 +66,7 @@
 AC_SUBST(LIBBLAM_LIBS)
 
 dnl Intl
-ALL_LINGUAS="bs ca cs da de el en_CA en_GB es eu fr ga he hr nl no pa pt_BR pt sq sr@Latn sr sv tr uk zh_CN"
+ALL_LINGUAS="bs ca cs da de el en_CA en_GB es eu fr ga he hr nl nb pa pt_BR pt sq sr@Latn sr sv tr uk zh_CN"
 GETTEXT_PACKAGE=blam
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE,"$GETTEXT_PACKAGE", [Gettext package])
