diff -urN aaa/blam-1.8.2/configure.in blam-1.8.2/configure.in
--- aaa/blam-1.8.2/configure.in	2005-06-20 17:12:07.000000000 +0200
+++ blam-1.8.2/configure.in	2006-08-14 20:30:29.777546250 +0200
@@ -51,14 +51,14 @@
 
 PKG_CHECK_MODULES(BASE_DEPENDENCIES, 
 		  mono >= $MONO_REQUIRED_VERSION
-		  gtk-sharp >= $GTKSHARP_REQUIRED_VERSION
+		  gtk-sharp-2.0 >= $GTKSHARP_REQUIRED_VERSION
                   gconf-2.0 >= $GCONF_REQUIRED_VERSION)
 
 PKG_CHECK_MODULES(BLAM,
-		  gtk-sharp >= $GTKSHARP_REQUIRED_VERSION
-                  gecko-sharp >= $GECKOSHARP_REQUIRED_VERSION
-		  gconf-sharp >= $GTKSHARP_REQUIRED_VERSION
-		  glade-sharp >= $GTKSHARP_REQUIRED_VERSION)
+		  gtk-sharp-2.0 >= $GTKSHARP_REQUIRED_VERSION
+                  gecko-sharp-2.0 >= $GECKOSHARP_REQUIRED_VERSION
+		  gconf-sharp-2.0 >= $GTKSHARP_REQUIRED_VERSION
+		  glade-sharp-2.0 >= $GTKSHARP_REQUIRED_VERSION)
 
 AC_SUBST(BLAM_LIBS)
 
diff -urN aaa/blam-1.8.2/src/Printing.cs blam-1.8.2/src/Printing.cs
--- aaa/blam-1.8.2/src/Printing.cs	2005-06-18 19:00:42.000000000 +0200
+++ blam-1.8.2/src/Printing.cs	2006-08-14 22:20:45.915029000 +0200
@@ -14,7 +14,7 @@
 namespace Imendio.Blam {
     public class PrintDialog {
 	private ItemView itemView;
-	private PrintJob printJob;
+	private Gnome.PrintJob printJob;
 
 	private static string printConfigFile = Application.BaseDir + "/print-config";
 
@@ -31,7 +31,7 @@
 		return;
 	    }
 	    
-	    printJob = new PrintJob(LoadPrintConfig());
+	    printJob = new Gnome.PrintJob(LoadPrintConfig());
 	    Gnome.PrintDialog dialog = new Gnome.PrintDialog(printJob,
 							     "Print - " + itemView.Widget.Title);
 	    dialog.Icon = Gdk.Pixbuf.LoadFromResource("blam.png");
